apps/condo/pages/auth/forgot.tsx

```diff
@@ -47,7 +47,7 @@ function ResetPageView () {
     const REGISTER_PHONE_LABEL = <label style={ { alignSelf:'end' } }>{PhoneMsg}</label>
 
     const [form] = Form.useForm()
-    const { executeRecaptcha } = useGoogleReCaptcha()
+    const { executeCaptcha } = useHCaptcha()
```

Comment:
Can we make HCaptcha optional? Our open-source users might not want to setup an additional dependency

Reply:
Updated

Now on the front-end the captcha can be without configuration or disabled. If this is so, then instead of the token the text `condo-captcha-disabled` will be sent (this is just a placeholder, there could be any other text here, it does not affect anything, just the token field is required in the API)

Everything on the server remains the same. If the `DISABLE_CAPTCHA` flag is set, then the captcha check will not work. If for some reason you have not specified the configuration, the corresponding warnings and errors will be displayed. Therefore, if you want to disable the captcha, then this must be done explicitly through the `DISABLE_CAPTCHA` flag